src/measure_theory/constructions/prod.lean

```diff
@@ -320,6 +320,10 @@ instance prod.measure_space {α β} [measure_space α] [measure_space β] : meas
 
 variables {μ ν} [sigma_finite ν]
 
+lemma volume_eq_prod (α β) [measure_space α] [measure_space β] :
```

Comment:
or `volume_prod`?

Reply:
The problem with `voume_prod` is that it can be interpreted as `volume of a product set`. I wanted to avoid this ambiguity. I can use `volume_eq_volume_prod_volume` if you prefer.

Reply:
`volume_eq_prod` is better then.